app/controllers/admin/slides_controller.rb

```diff
@@ -1,24 +1,26 @@
-class Admin::SlidesController < Admin::ApplicationController
-  def index
-    ransack_params = params[:q]
-    @q = Slide.search(ransack_params)
-    @slides = @q.result(distinct: true).
-              latest.
-              paginate(page: params[:page], per_page: 20)
-  end
+module Admin
+  class SlidesController < Admin::BaseController
+    def index
+      ransack_params = params[:q]
+      @q = Slide.search(ransack_params)
+      @slides = @q.result(distinct: true).
+        latest.
+        paginate(page: params[:page], per_page: 20)
+    end
 
-  def edit
-    @slide = Slide.find(params[:id])
-  end
+    def edit
+      @slide = Slide.find(params[:id])
+    end
 
-  def update
-    params.permit! # It's OK because of admin
-    @slide = Slide.find(params[:slide][:id])
-    @slide.assign_attributes(params[:slide])
-    if @slide.update_attributes(params[:slide])
-      redirect_to "/admin/slides/#{@slide.id}/edit"
-    else
-      render :edit
+    def update
+      params.permit! # It's OK because of admin
```

Comment:
__[brakeman]__
* Parameters should be whitelisted for mass assignment [:ref](http://brakemanscanner.org/docs/warning_types/mass_assignment/)

<!-- Comment by [SideCI](https://sideci.com) -->